js/src/term.js

```diff
@@ -221,11 +223,29 @@ jQuery(
 							}
 						);
 
-						// Modifies the text direction
-						$( 'body' ).removeClass( 'pll-dir-rtl' ).removeClass( 'pll-dir-ltr' ).addClass( 'pll-dir-' + dir );
+						// Creates an event once the language has been successfully changed.
+						const onTermLangChoice = new CustomEvent(
+							"onTermLangChoice",
+							{
+								detail: {
+									lang: JSON.parse( selectedOption.options[selectedOption.options.selectedIndex].getAttribute( 'data-lang' ) )
+								},
+							}
+						);
+						document.dispatchEvent( onTermLangChoice );
 					}
 				);
 			}
 		);
+
+		// Listen to `onTermLangChoice` to perform actions after the language has been changed.
+		document.addEventListener(
+			'onPostLangChoice',
```

Comment:
```suggestion
			'onTermLangChoice',
```
Am I missing something? 😅

Reply:
Nope, copy/paste.